start.sh

```diff
@@ -32,6 +26,7 @@ set -- "${POSITIONAL[@]-default}" # restore positional parameters
 SLEEP_TIME=${SLEEP_TIME:-15}
 echo "using sleep time of $SLEEP_TIME seconds, use -s or --sleep option to change"
 
+cp docker-compose-template.yml docker-compose.yml
```

Comment:
Waar is deze cp voor nodig? In regel 61 wordt docker-compose.yml overschreven via sed?

Reply:
The docker-compose.yml is removed from git (and .gitignored) to avoid the problem of accidentally committing a generated key. So, on a clean checkout, there is no docker-compose.yml. That's why it has to be copied first. As soon as the apiKey is generated, it can be replaced in the docker-compose.yml.
I have considered an alternative approach which is to pass the apiKey to docker-compose by means of an environment veriable. But that is much more cumbersome.